### DISCUSSION TITLE
Docs: Improve "Edit this Page" Workflow

### DISCUSSION THREAD

**Discussion by erondpowell** [directus/directus#13045] [Feature Requests]
At the bottom of our docs, we have an **Edit this page** option to let users open the docs markdown files and submit edits.
However, the current flow makes it harder for meaningful edits to be submitted. 

<img width="1017" alt="Screen Shot 2022-04-28 at 8 33 54 AM" src="https://user-images.githubusercontent.com/62306630/165764513-089d228d-7789-4f1a-8600-f616e4dc43c5.png">

When core team members use this, it allows us to create **and name** a branch on the directus/directus repo.

<img width="1537" alt="Screen Shot 2022-04-28 at 8 34 25 AM" src="https://user-images.githubusercontent.com/62306630/165765368-db672789-8dcd-4af9-969f-c9926de3c5cd.png">

However, when others try to make edits, it clones the repo and creates an auto-named branch "patch-N" that is saved on the submitter's fork of the repo.

<img width="1557" alt="Screen Shot 2022-04-28 at 8 32 58 AM" src="https://user-images.githubusercontent.com/62306630/165765072-9c1c5fae-fe89-470e-8937-920267680254.png">

I realize this doesn't get used *a lot* but in the current state, it also makes takes away the entire "quick and dirty"

**Comment by rijkvanzanten** [comment]
@erondpowell Doesn't the _Propose changes_ button open a PR? If that's the case, I'd argue this is actually quite a decent workflow 🤔

**Reply by benhaynes** [reply]
I thought so too, let's confirm... because I thought there was a pretty straightforward path by just editing -> PR.

**Reply by yumixiaoping** [reply]
Yes, it IS the way you guys described.... I made a hard stop in the user-flow before when I saw you couldn't name the branch.
It DOES make the branch, commits, PRs, etc.

(by the way: yumixiaoping = Eron)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
